client/src/lib/components/buy/buy-modal.svelte

```diff
@@ -28,6 +28,8 @@
       salePrice: toHexWithPadding(sellAmount),
       amountToStake: toHexWithPadding(stakeAmount),
       liquidityPoolAddress: selectedToken?.lpAddress || '',
+      tokenAddress: $selectedLandMeta?.tokenAddress || '',
+      currentPrice: $selectedLandMeta?.sellPrice || '',
```

Comment:
```suggestion
      liquidityPoolAddress: selectedToken?.lpAddress ?? '',
      tokenAddress: $selectedLandMeta?.tokenAddress ?? '',
      currentPrice: $selectedLandMeta?.sellPrice ?? '',
```
Consistency is key :)